Resources/Prototypes/Entities/Mobs/Species/diona.yml

```diff
@@ -35,6 +35,44 @@
         amount: 5
   - type: Bloodstream
     bloodReagent: Water
+  - type: Reactive
+    groups:
+      Flammable: [ Touch ]
+      Extinguish: [ Touch ]
+      Acidic: [Touch, Ingestion]
+    reactions:
+    - reagents: [Water, SpaceCleaner]
+      methods: [Touch]
+      effects:
+      - !type:WashCreamPieReaction
+    - reagents: [ PlantBGone ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        scaled: true
+        damage:
+          types:
+            Blunt: 7
+            Slash: 5
+            Piercing: 8
```

Comment:
This should not be doing anywhere near enough damage to kill in 5 shots. Waterguns shoot incredibly quickly and the cargo changes will make Plant-B-Gone incredibly available- Diona are in a bad enough spot balance wise as-is. Something akin to slime's water damage should be used.

Reply:
That's fair.
Dropped to 7 overall, (15 shot crit, cannot full kill)
- 2 Blunt
- 2 Slash
- 3 Piercing